flyway-core/pom.xml

```diff
@@ -210,6 +210,29 @@
             <artifactId>db2jcc4</artifactId>
             <scope>test</scope>
         </dependency>
+		<dependency>
+		    <groupId>org.neo4j</groupId>
+		    <artifactId>neo4j-ogm</artifactId>
+		    <version>2.1.3</version>
+		    <type>pom</type>
+		    <scope>test</scope>
+		</dependency>
+		<dependency>
+		    <groupId>org.neo4j</groupId>
+		    <artifactId>neo4j-configuration</artifactId>
+		    <version>3.2.3</version>
+		    <scope>test</scope>
+		</dependency>
+		<dependency>
```

Comment:
Please do not add any required dependencies. Flyway is a library with zero required transitive dependencies and we want it to stay that way.

Reply:
Cleaned transitive dependencies.

Reply:
No, they are still there...

Reply:
I removed the neo4j-configuration and neo4j-ogm from the pom file.